.github/workflows/build-publish.yaml

```diff
@@ -35,27 +42,43 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ github.repository }}
+          images: 
+            ${{ github.repository }}
+            ${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.GCP_PROJECT_ID }}/${{ env.GAR_REPOSITORY }}/${{ env.IMAGE }}
           # https://github.com/marketplace/actions/docker-metadata-action#tags-input
           tags: |
             type=semver,pattern={{raw}}
             type=raw,value=latest,enable={{is_default_branch}}
             type=sha,prefix=,enable={{is_default_branch}}
 
+      - id: gcp_auth
+        name: Log into GCP
+        uses: google-github-actions/auth@v2
+        if: github.event_name != 'pull_request'
+        with:
+          token_format: access_token
+          service_account: artifact-writer@${{ env.GCP_PROJECT_ID }}.iam.gserviceaccount.com
+          workload_identity_provider: ${{ vars.GCPV2_GITHUB_WORKLOAD_IDENTITY_PROVIDER }}
```

Comment:
This looks a bit different than:

https://github.com/mozilla-services/telescope/blob/ba8b8ac7914a039db41c8ca3436dbba3d65f2ec7/.github/workflows/build-publish.yaml#L63-L78

Is this the preferred way?
Should we align both?